app/assets/stylesheets/_theme.scss

```diff
@@ -25,7 +25,7 @@
   --items-column-border-left-color: var(--sn-stylekit-border-color);
   --items-column-border-right-color: var(--sn-stylekit-border-color);
   --items-column-search-background-color: var(--sn-stylekit-contrast-background-color);
-  --item-cell-selected-background-color: var(--sn-stylekit-contrast-background-color);
+  --item-cell-selected-background-color: var(--sn-stylekit-grey-5);
```

Comment:
Can we please phase out these grey colors 🙏 they haven’t made sense to since they were introduced. Just name them something else. Anything else.

Reply:
We’ll do this in the next PR. Please keep it in mind.